app/controllers/sign_up/recovery_codes_controller.rb

```diff
@@ -6,10 +6,7 @@ class RecoveryCodesController < ApplicationController
     before_action :confirm_has_not_already_viewed_recovery_code, only: [:show]
 
     def show
-      if user_session.delete(:first_time_recovery_code_view).present?
-        @show_progress_bar = true
-      end
-
+      user_session.delete(:first_time_recovery_code_view)
```

Comment:
was this key only used to decide whether or not to show the progress bar? if so, we can just remove it entirely right? (stop setting it and stop reading it?)